src/components/LaunchFilter/reducers/launchReducer.ts

```diff
@@ -0,0 +1,34 @@
+import { LaunchActions } from '../actions/actionTypesTemplates';
+import { LaunchFilters } from '../constants/filerConstants';
+import { FilterAction } from '../types/LaunchesAction';
+
+const initialState = {
+  [LaunchFilters.PREVIOUS]: [],
+  [LaunchFilters.CREWED]: [],
+  [LaunchFilters.CALIFORNIA]: [],
+  [LaunchFilters.FLORIDA]: [],
+  isLoading: false,
+  error: '',
+};
+
+export const launchReducer = (state = initialState, action: FilterAction) => {
+  if (!action.payload) return state;
+  const { filterValue, error } = action.payload;
+  switch (action.type) {
+    case `${filterValue}${LaunchActions.LAUNCH_REQUEST}`:
+      return { ...state, isLoading: true, error: '' };
```

Comment:
error is not needed here

Reply:
If I clear it, after success call to backend there are both results and error in the store. I think it's would be a mistake.
If the call to backend is rejected error will be rewritten.  Therefore I sppose error should be cleared here.

Reply:
Your initial state includes empty error field

Reply:
Discussed at the daily